apps/meteor/client/views/room/hooks/useOpenRoom.ts

```diff
@@ -25,7 +27,7 @@ export function useOpenRoom({ type, reference }: { type: RoomType; reference: st
 		// we need to add uid and username here because `user` is not loaded all at once (see UserProvider -> Meteor.user())
 		queryKey: ['rooms', { reference, type }, { uid: user?._id, username: user?.username }] as const,
 
-		queryFn: async (): Promise<{ rid: IRoom['_id'] }> => {
+		queryFn: async (): Promise<{ rid: IRoom['_id']; type?: IRoom['t'] }> => {
```

Comment:
Why are you returning type here? I don't see it being used anywhere. Maybe left by accident?